modules/crud2.js

```diff
@@ -0,0 +1,51 @@
+// reset todo id
+export const idReset = (lists) => {
+  lists.forEach((todo, ind) => {
+    todo.id = ind + 1;
+  });
+};
+
+// logic to delete all completed todos
+export const deleteCompletedTodos = (todoListss, todoUl) => {
+  const newTodos = todoListss.filter((item) => item.completed !== true);
+  idReset(newTodos);
+  localStorage.setItem('todos', JSON.stringify(newTodos));
+
+  const elements = [...todoUl.children];
+  elements.forEach((ele) => {
+    if (ele.id === 'completed') {
+      ele.remove();
+    }
+  });
+};
+
+// functionality to mark a todo as completed
+export const selectTodo = (checkBox, todoTitle, item, itemContainer) => {
+  itemContainer.id = 'completed';
+  if (checkBox.checked) {
+    todoTitle.style.textDecoration = 'line-through';
+    item.completed = true;
+  } else {
+    todoTitle.style.textDecoration = 'none';
+    item.completed = false;
+  }
```

Comment:
- [ ] As mentioned by the previous reviewer, kindly make sure the task object's values for `completed` is updated upon user actions i.e. when clicking the `checkbox` button from your UI, it should update at the same the task object's values for `completed` in the local storage according to the current value. Right now from your app it remains the same status. 
![comp](https://user-images.githubusercontent.com/83042742/214796437-714a380b-5637-42be-a4fa-410c71119cf2.gif)